knimin/lib/configuration.py

```diff
@@ -103,5 +104,9 @@ def _get_qiita(self, config):
         self.qiita_client_secret = config.get('qiita', 'CLIENT_SECRET')
         self.qiita_server_cert = config.get('qiita', 'SERVER_CERT')
 
+    def _get_platemapper(self, config):
+        self.pm_sample_sheet_dir = config.get('platemapper',
+                                              'SAMPLE_SHEET_DIR')
```

Comment:
oh, sorry, I was looking at this incorrectly. This is for configuration retrieval. Please disregard.